writeup.md

```diff
@@ -0,0 +1,18 @@
+# Analysis
+
+#### Nick Dixon, dixonnj@msoe.edu
```

Comment:
You didn't include the picture of the code coverage as was stated in the writeup.